src/auth-service/config/dbConnection.js

```diff
@@ -1,24 +1,51 @@
 const mongoose = require("mongoose");
+// const { Gstore, instances } = require("gstore-node");
 const config = require("./constants");
+// const { Datastore } = require("@google-cloud/datastore");
+
+// const gstore1 = new Gstore();
+// const gstore2 = new Gstore({ cache: true });
+// const gstore3 = new Gstore();
+
+// const datastore1 = new Datastore({
+//   namespace: "default",
+//   projectId: "airqo-250220",
+// });
+// const datastore2 = new Datastore({
+//   namespace: "cache-on",
+//   projectId: "airqo-250220",
+// });ç
+// const datastore3 = new Datastore({
+//   namespace: "staging",
+//   projectId: "airqo-250220",
+// });
```

Comment:
These comments still exist because I am reserving the code for the migration towards Datastore in the near future. Otherwise, I usually discourage comments in code.